docs/guides/training_techniques/batch_norm.rst

```diff
@@ -79,17 +85,17 @@ The ``batch_stats`` collection must be extracted from the ``variables`` for late
   :title_right: With BatchNorm
   :sync:
 
-  mlp = MLP()
+  model = Model()
   x = jnp.ones((1, 3))
-  variables = mlp.init(jax.random.key(0), x)
+  variables = model.init(jax.random.key(0), x)
   params = variables['params']
 
 
   jax.tree_util.tree_map(jnp.shape, variables)
   ---
-  mlp = MLP()
+  bn_model = BatchNormModel()
```

Comment:
can we keep names the same as much as possible? I think it helps visually when you change tabs that only the actual differences in code change.

Reply:
The problem is that `model` is referenced in later code snippets, e.g.: https://flax--3847.org.readthedocs.build/en/3847/guides/training_techniques/batch_norm.html#training-and-evaluation
So if we use `model` for both Haiku and Flax, Flax would overwrite the `model` and the next Haiku code snippet would be using the Flax `model` instead of the Haiku `model`.